src/components/Footer/index.js

```diff
@@ -9,30 +9,42 @@ import {
 
 import { ContactCard } from '../ContactCard';
 
-export function Footer({ email, tel, street, city, zip }) {
+export function Footer({ email, offices }) {
 	return (
 		<div>
 			<footer className="pv3 pv5-m bg-black white">
 				<div className="grid items-center">
-					<div className="grid-item w-10 push-1 w-7-l mb2 mb0-l">
-						<ContactCard
-							className="mb1 mb0-m light-grey"
-							tel={tel}
-							email={email}
-							street={street}
-							city={city}
-							zip={zip}
-						/>
+					<div className="grid-item flex-m w-10 push-1 w-7-l mb2 mb0-l">
+						{offices.map(office => (
```

Comment:
I would deconstruct here to tidy it up a little 

```suggestion
						{offices.map(({ name, tel, street, city, zip}) => (
```